QuickStart.md

```diff
@@ -16,41 +79,34 @@ hmily-admin含有**两种部署方式**：`release`方式、`docker`方式，您
 
 #### 步骤一：获取release包文件
 
-> 方法一：直接获取
-
-​    release包已放在`hmily-admin/hmily-admin-service/target/hmily-admin-1.0.2.zip`，获取地址见：[hmily-admin-1.0.2.zip](https://github.com/dromara/hmily-admin/tree/master/hmily-admin-service/target/hmily-admin-1.0.2.zip)
-
-> 方法二：maven打包生成
-
-- 在项目目录下，使用maven进行打包，生成target目录。
+获取地址见：[hmily-admin-1.0.2-admin-bin.tar.gz](https://github.com/dromara/hmily-admin/releases/download/1.0.2/)
```

Comment:
It's not a good idea to leave a URL with a specified version here.  You have to update the version every time.